pom.xml

```diff
@@ -148,7 +148,7 @@
     <vers.hbase2>0.98.3-hadoop2</vers.hbase2>
     <vers.hbase-cdh4>0.94.6-cdh${cdh4.version}</vers.hbase-cdh4>
     <vers.hbase-cdh5>0.96.1.1-cdh${cdh5.version}</vers.hbase-cdh5>
-    <vers.hive>0.10.0</vers.hive>
+    <vers.hive>0.13.0</vers.hive>
```

Comment:
"STORED AS PARQUET" was added in Hive 0.13, so it makes sense to update the dependency for that reason. "STORED AS AVRO" is in Hive 0.14.

Until we hit 1.0 we should track dependencies more aggressively, IMO. For backwards compatibility in CDH4 - that should be done in CDH itself.